libbeat/filter/filter.go

```diff
@@ -65,6 +65,11 @@ func (filters *FilterList) Get(index int) FilterRule {
 // Applies a sequence of filtering rules and returns the filtered event
 func (filters *FilterList) Filter(event common.MapStr) common.MapStr {
 
+	// Check if filters are set, just return event if not
+	if filters == nil {
```

Comment:
😕  It's not possible to call this method if the receiver is `nil`, right? This check seems unnecessary.